hudi-timeline-service/src/main/java/org/apache/hudi/timeline/service/handlers/marker/MarkerBasedEarlyConflictDetectionRunnable.java

```diff
@@ -109,7 +109,7 @@ public void run() {
       currentFileIDs.retainAll(tableFilesIDs);
       if (!currentFileIDs.isEmpty()
           || (checkCommitConflict && MarkerUtils.hasCommitConflict(activeTimeline,
-          currentInstantAllMarkers.stream().map(MarkerUtils::makerToPartitionAndFileID).collect(Collectors.toSet()), completedCommits))) {
+          currentInstantAllMarkers.stream().map(MarkerUtils::makerToFileID).collect(Collectors.toSet()), completedCommits))) {
```

Comment:
It's hard to create a scenario described in rfc-56, `Writer1 starts writing data at time t1 and finishes at time t3. Writer2 starts writing at time t2, and at time t4 - writer2 tries to create marker for a file fileA which already updated by writer1 to fileAa(updated)`.

But to be consistent with  MarkerUtils.hasCommitConflict, the 2nd param should be a set of fileId.